lib/ruby_lsp/requests/code_action_resolve.rb

```diff
@@ -0,0 +1,105 @@
+# typed: strict
+# frozen_string_literal: true
+
+module RubyLsp
+  module Requests
+    # ![Code action resolve demo](../../misc/code_action_resolve.gif)
+    #
+    # The [code action resolve](https://microsoft.github.io/language-server-protocol/specification#codeAction_resolve)
+    # request is used to to resolve the edit field for a given code action, if it is not already provided in the
+    # textDocument/codeAction response. We can use it for scenarios that require more computation such as refactoring.
+    #
+    # Currently one use case is implemented: Extract to variable.
+    #
+    # # Example
```

Comment:
If we say that we only support one use case, there's a very high chance we'll forget to update this phrase once more are supported 😅.
```suggestion
    # # Extract to variable example
```